main.go

```diff
@@ -200,61 +287,14 @@ func main() {
 			}
 		}
 	}
+	return publicInstallPages, nil
+}
 
-	fmt.Println()
-	log.Donef("Success")
-	log.Printf("You can find the Artifact on Bitrise, on the Build's page: %s", config.BuildURL)
-
-	if len(publicInstallPages) > 0 {
-		temp := template.New("Public Install Page template")
-		var pages []PublicInstallPage
-		for file, url := range publicInstallPages {
-			pages = append(pages, PublicInstallPage{
-				File: file,
-				URL:  url,
-			})
-		}
-
-		if err := tools.ExportEnvironmentWithEnvman("BITRISE_PUBLIC_INSTALL_PAGE_URL", pages[0].URL); err != nil {
-			fail("Failed to export BITRISE_PUBLIC_INSTALL_PAGE_URL, error: %s", err)
-		}
-		log.Printf("The public install page url is now available in the Environment Variable: BITRISE_PUBLIC_INSTALL_PAGE_URL (value: %s)\n", pages[0].URL)
-
-		temp, err := temp.Parse(config.PublicInstallPageMapFormat)
-		if err != nil {
-			fail("Error during parsing PublicInstallPageMap: ", err)
-		}
-
-		buf := new(bytes.Buffer)
-		if err := temp.Execute(buf, pages); err != nil {
-			fail("Execute: ", err)
-		}
-
-		if err := tools.ExportEnvironmentWithEnvman("BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP", buf.String()); err != nil {
-			fail("Failed to export BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP, error: %s", err)
-		}
-		log.Printf("A map of deployed files and their public install page urls is now available in the Environment Variable: BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP (value: %s)", buf.String())
-		log.Printf("")
-	}
-
-	// Deploy test files
-	if config.AddonAPIToken != "" {
-		fmt.Println()
-		log.Infof("Upload test results")
-
-		testResults, err := test.ParseTestResults(config.TestDeployDir)
-		if err != nil {
-			log.Warnf("Error during parsing test results: ", err)
-		} else {
-			log.Printf("- uploading (%d) test results", len(testResults))
-
-			if err := testResults.Upload(config.AddonAPIToken, config.AddonAPIBaseURL, config.AppSlug, config.BuildSlug); err != nil {
-				log.Warnf("Failed to upload test results: ", err)
-			} else {
-				log.Donef("Success")
-			}
-		}
+func getExtension(pth string) string {
+	if xcarchive.CheckForXcarchive(pth) {
+		return zippedXcarchiveExt
```

Comment:
this is not an extension, .zip would be, .xcarchive.zip is a zip file with .xcarchive suffix

Reply:
Renamed.